### ISSUE TITLE
IPv6: Rewrite IP assignment

### ISSUE THREAD

**Issue by schiederme** [zeug/openvm#1]
To create IPv6 Support, the script for assignment needs to be rewritten.

**Comment by schiederme** [comment]
See https://github.com/subuk/vmango/issues/3 for reference.

**Comment by mraerino** [comment]
First thing:
How do you want to assign addresses with IPv6? Any reasons against slaac in the machine?

**Comment by schiederme** [comment]
Nope. I currently use SLAAC on the test server. The problem is that Vmango dose not fully create the vm, when IPv6 is active.

**Comment by schiederme** [comment]
I don't understand why Vmango is not booting/creating the vm fully.

**Comment by mraerino** [comment]
So is the ipv4 assignment blocking because you are trying v6-only?

**Comment by schiederme** [comment]
No. I try to assign both v4 and v6. As of my understanding, the VM currently get a v6.

**Comment by schiederme** [comment]
But I can't control it from the panel.

**Comment by mraerino** [comment]
Ok. I need to understand the larger scheme of VM creation and IP assignment a little better I think. Let me drive into the specifics and maybe I can try to propose a fix.

**Comment by schiederme** [comment]
Thank you. I maybe have an idea what causes the problem. I will try something and will come back later/tomorrow.

**Comment by schiederme** [comment]
OK. Subuk has patched some things in to Vmango (bridgednet branch) which allows to use a different DHCP server. So you can easily fix this problem by using something like iscdhcp instead of dnsmasq. 

I will try this tomorrow/today and will give you an update asap.
